input/demos/aragog.toml

```
@@ -92,7 +94,7 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
     rheo_phi_loc    = 0.4    # Centre of rheological transition
     rheo_phi_wid    = 0.15   # Width of rheological transition
     bulk_modulus    = 260e9   # Bulk modulus [Pa]
-
+    melting_dir     ="Monteaux-600"
```

```suggestion
    melting_dir     = "Monteux-600"
```
Also can you please add a comment in this line.